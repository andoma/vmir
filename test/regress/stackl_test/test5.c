
int main()
{
    int ii;
    ii = 0;
    while (ii < 3)
    {
        puts("while loop\n");
        ii = ii + 1;
    }
}
