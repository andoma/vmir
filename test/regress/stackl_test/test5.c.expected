while loop
while loop
while loop
